### PR TITLE
error message in assertion for GroupTest updated

### PR DESCRIPTION
**Pull request by S-Panta**
Related issue:https://github.com/owncloud/ocis-php-sdk/issues/131
Ocis master and stable is bumped in this PR

### CODE REVIEW

File: tests/integration/Owncloud/OcisPhpSdk/GroupsTest.php [hunk 1]
```diff
@@ -158,9 +179,19 @@ public function testDeleteGroup(): void
...
         $this->createdGroups = [$physicsLoversGroup];
+        $this->assertCount(
+            1,
+            $ocis->getGroups(),
+            "Expected one number of group but got ". count($ocis->getGroups())
+        );
+        $this->assertEquals(
+            "physicslovers",
+            $ocis->getGroups()[0]->getDisplayName(),
+            "Group name should be physicslovers but found "
```
Comment (amrita-shrestha) [inline tests/integration/Owncloud/OcisPhpSdk/GroupsTest.php:191]:
```suggestion
            "Group should be deleted but exists "
```

File: tests/integration/Owncloud/OcisPhpSdk/GroupsTest.php [hunk 2]
```diff
@@ -173,9 +204,18 @@ public function testGetGroupsByNormalUser(): void
         $this->createdGroups = [$philosophyHatersGroup];
         $marieOcis = $this->getOcis('marie', 'radioactivity');
         $groups = $marieOcis->getGroups("philosophyhaters");
-        $this->assertCount(1, $groups);
+        $this->assertCount(
+            1,
+            $groups,
+            "Expected one number of group but got ". count($groups)
```
Comment (amrita-shrestha) [inline tests/integration/Owncloud/OcisPhpSdk/GroupsTest.php:210]:
```suggestion
            "Expected one group but found ". count($groups)
```

**Review by amrita-shrestha — COMMENTED** [review]
address this review

**Review by amrita-shrestha — APPROVED** [review]
lgtm

File: tests/integration/Owncloud/OcisPhpSdk/GroupsTest.php [hunk 3]
```diff
@@ -26,8 +26,19 @@ public function testAddUserToGroup(): void
...
+                1,
+                $group->getMembers(),
+                "The group ".$group->getDisplayName()
+                . " should have 1 member but found "
+                .count($group->getMembers())." members"
+            );
+            $this->assertEquals(
+                $userName,
+                $group->getMembers()[0]->getDisplayName(),
+                $userName . " user should exist in ". $group->getDisplayName() . " group but found "
```
Comment (individual-it) [inline tests/integration/Owncloud/OcisPhpSdk/GroupsTest.php:39]:
```suggestion
                $userName . " user be the first membe in ". $group->getDisplayName() . " group but found "
```

**Review by individual-it — CHANGES_REQUESTED** [review]

**Review by individual-it — APPROVED** [review]

**Pull request merged**